.travis.yml

```diff
@@ -17,12 +17,12 @@ jobs:
   include:
     - stage: deploy
       node_js: stable
-      script: skip
+      script: npm run build
```

Comment:
I reverted this and created a `scripts.prepublish` in the package.json with `npm run build` there. That way a manual `npm publish` also compiles before publishing.